firebase/functions/index.js

```diff
@@ -0,0 +1,24 @@
+const functions = require('firebase-functions');
+const sha1 = require('sha1');
+const format = require("date-format-lite")
```

Comment:
do you need this package ?

Reply:
- firebase-functions to use cloud-functions
- sha1 - to encrypt the token
- format - to format the date